test/internet/test-dns.js

```diff
@@ -471,6 +471,42 @@ TEST(function test_lookup_localhost_ipv4(done) {
 });
 
 
+TEST(function test_lookup_all_ipv4(done) {
```

Comment:
No. If you don't specify the family, you should be able to get all IPv4 and IPv6 addresses.